docs/blog/newsletter-october-2019.rst

```diff
@@ -0,0 +1,116 @@
+* Establish some clear, short-term deliverables for your work, to get some early wins and regain confidence.
+* Remember that you took time to learn the things you already know, so don't feel discouraged when you feel like you can't grasp new material instantly. This work is difficult! Step-by-step, you'll eventually understand the product and get the knowledge you need.
+* Make a list of skills and knowledge that you feel you're missing, and then work out what you'd need to do to fill the gaps. Or, make a list of specific, concrete things you want to achieve, have achieved, expect to achieve. Involve your manager so you can turn them into actionable goals. Success is a work in progress.
+
+  If you're interested in setting clear actionable objectives to improve your performance, check out the `SMART criteria <https://en.wikipedia.org/wiki/SMART_criteria>`_.
+
+-------------------------------------------------------
+Leading and following: finding a mentor, being a mentor
+-------------------------------------------------------
+
+Another great way to build both your confidence and your abilities is to find a mentor, as we discussed recently in  `#career-advice`. General agreement that having a mentor is A Very Good Idea led to some handy info:
+
+* In the UK, the ISTC has a lightweight `mentoring program <https://www.istc.org.uk/professional-development-and-recognition/mentoring-scheme/>`__. One program participant explained that there aren't many guidelines, so they set up a development plan with their mentee and gave feedback on the mentee's CV.
+* It can help to draft a development plan to show to a potential mentor, so they know what you're looking for. For ideas, see the ISTC's `Example CPD Records <https://www.istc.org.uk/professional-development-and-recognition/continuing-professional-development/example-cpd-records/>`_.
+* Some companies provide formal mentoring programs: people who want to mentor sign up and describe their expertise, and you contact them if you want to be mentored. These programs typically provide internal resources for mentors and mentees as well.
+* A useful article: `How to Email a Potential Mentor the Right Way <https://www.themuse.com/advice/be-my-mentor-craft-the-perfect-email-to-someone-you-admire>`__.
+
+A couple of contributors suggested that WTD start its own mentoring program, which sounds like an idea we'd love someone to take on! And if this topic captures your interest, see Arran Southall's talk from Prague this year, `Fostering Talent: Mentorship, Peer Reviews and Going Beyond Your Job Description <https://www.youtube.com/watch?v=rom6UW-TjNc&list=PLZAeFn6dfHpkpYchP1iFnQnc7i-2xJd0I&index=18>`__.
+
+(Side note: One *could* argue that we shouldn't use the term "mentee". Classically speaking, the right word for someone being guided by a Mentor is, of course, a "`Telemachus <https://en.wikipedia.org/wiki/Mentor_(Odyssey)#Mentor_as_term>`__".)
```

Comment:
I hope this should be obviously tongue-in-cheek, but please let me know if it doesn't come across that way!

Reply:
I think it comes across that way fine -- this is certainly the audience for it. 😉